LFPy/cell.py

```
@@ -543,6 +543,7 @@ def _set_extracellular(self):
 
         for sec in self.allseclist:
             sec.insert('extracellular')
+        self.extracellular = True
```

This overrides the existing `Cell.extracellular` attribute set in the `__init__` method. Perhaps I'd like better to raise an error if `self.extracellular != True` here instead?

---

I don't know. I guess that if a user explicitly calls `enable_extracellular_stimulation` then it should just set it to True as that is the intended use. Right?